mule-packager/src/main/java/org/mule/tools/api/packager/builder/PackageBuilderFactory.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Mule ESB Maven Tools
+ * <p>
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * <p>
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.tools.api.packager.builder;
+
+import org.mule.tools.api.packager.packaging.PackagingOptions;
+import org.mule.tools.api.packager.packaging.PackagingType;
+
+import static org.mule.tools.api.packager.packaging.PackagingType.MULE_DOMAIN_BUNDLE;
+
+/**
+ * Factory for package builders.
+ */
+public class PackageBuilderFactory {
```

Comment:
Please add a unit test for this class 
the idea here is to validate that baed on the projectInformation the returned object is of the correspondent type